src/app/zones/locales/en-us/index.yaml

```diff
@@ -1,29 +1,47 @@
 zone-cps:
   routes:
     item:
-      title: "{name} Zone CP"
-      breadcrumbs: Zone CPs
+      title: "{name} Control Plane"
+      breadcrumbs: Control Planes
       config:
         no-subscriptions: This zone has no subscriptions
     items:
-      title: Zone CPs
-      breadcrumbs: Zone CPs
+      title: Control Planes
+      breadcrumbs: Control Planes
+      headers:
+        name: 'Name'
+        zoneCpVersion: 'Zone CP Version'
+        storeType: 'Storage type'
+        status: 'Status'
+        warnings: 'Warnings'
+        actions: 'Actions'
```

Comment:
`actions:` should come from `common`

Reply:
Not moving this. Table headers should come from one place even though this might be duplicated.

Reply:
All actions columns for all collections should reference the same piece of text. We don't want to have potentially 10s of places in the i18n files that just repeat `Actions`. Please pull this from common.

Reply:
I don’t want to fragment the source of column names so I’m not moving them. Feel free to file a follow up PR if you want to do this differently.